zero/client_server/worker.py

```diff
@@ -89,24 +88,28 @@ def handle_msg(self, rpc, msg):
             return "connected"
 
         if rpc not in self._rpc_router:
-            logging.error(f"method `{rpc}` is not found!")
+            logging.error("method `%s` is not found!", rpc)
```

Comment:
f string is faster?

Reply:
https://github.com/Ananto30/zero/pull/34#discussion_r1160584800